addon/serializer.js

```diff
@@ -0,0 +1,37 @@
+import Model from './orm/model';
+import Collection from './orm/collection';
+import { pluralize } from './utils/inflector';
+import extend from './utils/extend';
+
+class Serializer {
+
+  serialize(response, request) {
+    if (response instanceof Model) {
+      return this.serializeModel(response);
+
+    } else if (response instanceof Collection) {
+      return this.serializeCollection(response);
+
+    } else {
+      return response;
+    }
+  }
+
+  serializeModel(model) {
+    return this.constructor.root ? { [model.type]: model.attrs } : model.attrs;
```

Comment:
If you override in a subclass, see the [test](https://github.com/samselikoff/ember-cli-mirage/pull/207/files#diff-13e6bf460e46a3755ef874d2806b55e0R68) below